email_notification/helpers/email_translations.py

```diff
@@ -0,0 +1,320 @@
+# ruff: noqa: E501
+from config.utils.commons import LanguageType
+
+EMAIL_TRANSLATIONS: dict[str, dict[LanguageType, str]] = {
+    # APPLICATION_HANDLED
+    "subjectApplicationHandled": {
```

Comment:
Why the `camelCasing`?

Reply:
Holdover from frontend version?

Reply:
Holdover from frontend, and to make it easier to differentiate translation strings vs variables